### PR TITLE
Feature/dininghalls

### PR DESCRIPTION
**Pull request by celinebrass**
Added in dining hall models.

Named branch correctly.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,34 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
```
Comment (ifilonenko) [inline db/seeds.rb]: Your seed file should only be visible to you. So its going to be sitting locally, you dont need to push it

File: db/migrate/20160228004231_add_location_to_dininghalls.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddLocationToDininghalls < ActiveRecord::Migration
```
Comment (ifilonenko) [inline db/migrate/20160228004231_add_location_to_dininghalls.rb]: Consolidate all migrations into one migration please, lots of migration files is tedious and unnecessary. your creation migration should handle all of these fields. Create a migration later one for additional fields (or SQL rows) you wish to add

File: db/migrate/20160228004104_add_location_short_to_dininghalls.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddLocationShortToDininghalls < ActiveRecord::Migration
```
Comment (ifilonenko) [inline db/migrate/20160228004104_add_location_short_to_dininghalls.rb]: delete these migrations because they are redundant

**Comment by ifilonenko** [comment]
Ready to ship!

**Comment by celinebrass** [comment]
8th time is the charm

Can I merge it or do you do that?

**Pull request merged**